packages/manager/apps/pci-rancher/src/components/GuideLink/guide-utils.tsx

```diff
@@ -0,0 +1,90 @@
+    ASIA:
+      'https://help.ovhcloud.com/csm/asia-public-cloud-managed-rancher-service-getting-started?id=kb_article_view&sysparm_article=KB0061896',
+    AU:
+      'https://help.ovhcloud.com/csm/en-au-public-cloud-managed-rancher-service-getting-started?id=kb_article_view&sysparm_article=KB0061910',
+    CA:
+      'https://help.ovhcloud.com/csm/en-ca-public-cloud-managed-rancher-service-getting-started?id=kb_article_view&sysparm_article=KB0061904',
+    DE:
+      'https://help.ovhcloud.com/csm/de-public-cloud-managed-rancher-service-getting-started?id=kb_article_view&sysparm_article=KB0061897',
+    ES:
+      'https://help.ovhcloud.com/csm/es-es-public-cloud-managed-rancher-service-getting-started?id=kb_article_view&sysparm_article=KB0061901',
+    EU:
+      'https://help.ovhcloud.com/csm/en-ie-public-cloud-managed-rancher-service-getting-started?id=kb_article_view&sysparm_article=KB0061898',
+    FR:
+      'https://help.ovhcloud.com/csm/fr-public-cloud-managed-rancher-service-getting-started?id=kb_article_view&sysparm_article=KB0061903',
+    GB:
+      'https://help.ovhcloud.com/csm/en-gb-public-cloud-managed-rancher-service-getting-started?id=kb_article_view&sysparm_article=KB0061909',
+    IE:
+      'https://help.ovhcloud.com/csm/en-ie-public-cloud-managed-rancher-service-getting-started?id=kb_article_view&sysparm_article=KB0061898',
+    IN:
+      'https://help.ovhcloud.com/csm/asia-public-cloud-managed-rancher-service-getting-started?id=kb_article_view&sysparm_article=KB0061896',
+    IT:
+      'https://help.ovhcloud.com/csm/it-public-cloud-managed-rancher-service-getting-started?id=kb_article_view&sysparm_article=KB0061907',
+    MA:
+      'https://help.ovhcloud.com/csm/fr-public-cloud-managed-rancher-service-getting-started?id=kb_article_view&sysparm_article=KB0061903',
+    NL:
+      'https://help.ovhcloud.com/csm/en-nl-documentation-public-cloud?id=kb_browse_cat&kb_id=574a8325551974502d4c6e78b7421938',
+    PL:
+      'https://help.ovhcloud.com/csm/pl-public-cloud-managed-rancher-service-getting-started?id=kb_article_view&sysparm_article=KB0061908',
+    PT:
+      'https://help.ovhcloud.com/csm/pt-public-cloud-managed-rancher-service-getting-started?id=kb_article_view&sysparm_article=KB0061905',
+    QC:
+      'https://help.ovhcloud.com/csm/fr-ca-public-cloud-managed-rancher-service-getting-started?id=kb_article_view&sysparm_article=KB0061899',
+    SG:
+      'https://help.ovhcloud.com/csm/en-sg-public-cloud-managed-rancher-service-getting-started?id=kb_article_view&sysparm_article=KB0061895',
+    SN:
+      'https://help.ovhcloud.com/csm/fr-public-cloud-managed-rancher-service-getting-started?id=kb_article_view&sysparm_article=KB0061903',
+    TN:
+      'https://help.ovhcloud.com/csm/fr-public-cloud-managed-rancher-service-getting-started?id=kb_article_view&sysparm_article=KB0061903',
+    US: 'https://us.ovhcloud.com/support',
```

Comment:
**question:** Is it the right link?

Reply:
Yes i got this link from the ticket description and i  rechecked with Alizée and Elena and they confirmed that it's correct since that rancher doesn't exist in the US yet
<img width="1205" alt="q1" src="https://github.com/ovh/manager/assets/30338862/a3e59d95-25a1-4a99-8a88-cc3bcf527690">